normandy/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+var path = require('path');
```

Comment:
Most projects I've seen put this at the root of the git repo. Is there a reason to put it in the `normandy` subdirectory here?